clippy.toml

```diff
@@ -0,0 +1 @@
+doc-valid-idents = ["DoughnutApi"]
```

Comment:
`DoughnutApi` is a rust trait, in this case so should be allowed

Reply:
sounds good